identity/src/main/kotlin/net/thechance/identity/api/controller/ProfileController.kt

```diff
@@ -0,0 +1,24 @@
+package net.thechance.identity.api.controller
+
+import net.thechance.identity.api.dto.ProfileResponse
+import net.thechance.identity.service.UserService
+import org.springframework.http.ResponseEntity
+import org.springframework.security.core.context.SecurityContextHolder
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import java.util.*
+
+@RestController
+@RequestMapping("/profile")
+class ProfileController(
+	private val userService: UserService,
+) {
+
+	@GetMapping("/me")
+	fun getCurrentUserProfile(): ResponseEntity<ProfileResponse> {
+		val userId = SecurityContextHolder.getContext().authentication.let { UUID.fromString(it.name) }
+		val response = userService.findById(userId)
+		return ResponseEntity.ok(response)
```

Comment:
you can get the user ID this way
```suggestion
	fun getCurrentUserProfile(@AuthenticationPrincipal userId: UUID): ResponseEntity<ProfileResponse> {
		val response = userService.findById(userId)
		return ResponseEntity.ok(response)
```

Reply:
Thanks @devfalah, I used this method it looks more cleaner.